### PR TITLE
Docs/change url for cloning (Issue #779)

### PR DESCRIPTION
**Pull request by sadamhussain-m**
I had the committed the changes on the README file for the above issue.I hope my commit resolves the issue.If not,please let me know for any further changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,7 +231,7 @@ cp .env.example .env
 
 The following environment variables must be set:
 
-* `NEXTAUTH_URL`
+* `NEXTAUTH_URL`
```
Comment (dguyen) [inline README.md:234]: Remove the whitespace

File: README.md [hunk 2]
```diff
@@ -115,10 +115,12 @@ To run Documenso locally, you will need
 
 Want to get up and running quickly? Follow these steps:
 
-1. [Clone the repository](https://help.github.com/articles/cloning-a-repository/) it to your local device.
+1. [Fork this repository ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) to your github account.
+
+After forking clone it using the below command:
```
Comment (dguyen) [inline README.md:120]: Could you please change this (and the one on line 159) to:

> After forking the repository, clone it to your local device by using the following command:

File: README.md [hunk 3]
```diff
@@ -115,10 +115,12 @@ To run Documenso locally, you will need
 
 Want to get up and running quickly? Follow these steps:
 
-1. [Clone the repository](https://help.github.com/articles/cloning-a-repository/) it to your local device.
+1. [Fork this repository ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) to your github account.
```
Comment (dguyen) [inline README.md:118]: Could you please change this (and the one on line 157) to:

```
1. [Fork this repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) to your GitHub account.
```

**Review by dguyen — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sadamhussain-m) [reply README.md:120]: Sure,I'll remove it.

File: README.md (hunk 3 above)
Comment (sadamhussain-m) [reply README.md:118]: I'll make the necessary changes for it and raise the PR again.

**Comment by sadamhussain-m** [comment]
Could you please review the changes now?.

File: README.md (hunk 2 above)
Comment (dguyen) [reply README.md:120]: It doesn't appear that this line has been updated yet?

> After forking clone it using the below command:

Comment (sadamhussain-m) [reply README.md]: I had updated the changes.Can you tell me is it fine?.

**Review by Mythie — APPROVED** [review]

**Comment by Mythie** [comment]
Thanks Sadam! We appreciate the changes and wish you a happy first contribution to Documenso 🎉 

thanks-adventure-time

**Comment by sadamhussain-m** [comment]
@Mythie Thanks a lot for accepting my PR.

**Pull request merged**